aten/src/ATen/native/native_functions.yaml

```diff
@@ -2680,3 +2680,252 @@
 - func: nonzero(Tensor self) -> Tensor
   variants: method, function
   device_guard: false
+
+- func: gather_out(Tensor result, Tensor self, int64_t dim, Tensor index) -> Tensor
+  device_guard: false
```

Comment:
I must have missed this previous diffs. `device_guard` is false, because the inner call will handle setting the device guard? We will need to remember to remove these when we actually port these functions.

Reply:
Yes, that is correct and noted here: https://github.com/pytorch/pytorch/pull/13003/commits/6c2d843195b75a2fc03e102cd283a8673098db36.